index.html

```diff
@@ -9,27 +9,32 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 
     <link rel="stylesheet" href="./index.css" />
+    <script defer src="./index.js"></script>
     <title>Toyyib</title>
   </head>
   <body>
 
     <div class="wrapper">
+
         <div class="top-container">
     
             <div class="header">
     
                 <a href="#" class="left-text">My Logo</a>
     
-                <nav class="right-hamburger">
+                <nav class="right-hamburger" id="hamburger">
     
-                    <img src="./assets/Union.png" alt="">
+                    <img src="./assets/Union.png" alt="" >
     
                 </nav>
 
                 <nav class="nav-li">
-                    <a href="#">Portfolio</a>
-                    <a href="#">About</a>
-                    <a href="#">Contact</a>
+                    <div class="x-men"><span>X</span></div>
+
+                    <a href="#portfolio" class="lis active">Portfolio</a>
+                    <a href="#about" class="lis">About</a>
+                    <a href="#contact" class="lis">Contact</a>
+
                 </nav>
```

Comment:
- [ ] please make sure that you pop-up window looks similar to the one in de figma template provided by Microverse:
Your project:
![afbeelding](https://user-images.githubusercontent.com/39240820/233052025-44558cc7-1c1a-4fa8-8500-b4a952310b29.png)
Figma template:
![afbeelding](https://user-images.githubusercontent.com/39240820/233052371-28e3e650-3359-488f-b600-17b9eed6d8a9.png)